DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -1,7 +1,7 @@
 {

```

Comment:
Can we remove this file from the PR, please, Manish

Reply:
done